api/src/test/java/org/openmrs/module/fhir2/api/search/EncounterSearchQueryTest.java

```diff
@@ -1539,4 +1539,21 @@ public void shouldReturnCollectionOfEncountersSortedByEncounterDate() {
 			    greaterThanOrEqualTo(resultList.get(i).getPeriod().getStart()));
 		}
 	}
+	
+	@Test
+	public void searchForEncounters_shouldReturnAllNonVoidedEncountersSortedById() {
```

Comment:
FWIW, this test was passing prior to this commit, but it does codify the behavior we now expect to see, that it sorts by id.